.github/workflows/go.yml

```diff
@@ -455,6 +455,59 @@ jobs:
           path: |
             /tmp/*.log
 
+  smoketest-airgap:
+    name: Smoke test for airgap install
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get PR Reference and Set Cache Name
+        run: |
+          PR_NUMBER=$(echo ${GITHUB_REF} | cut -d / -f 3 )
+          echo "cachePrefix=k0s-${PR_NUMBER}-${{ github.sha }}" >> $GITHUB_ENV
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Bindata
```

Comment:
Why do we need the embedded-bins cache for smoke? We should already have the full k0s bin pre-build and restored from cache, right?